nflow-engine/src/main/resources/scripts/db/db2.create.ddl.sql

```diff
@@ -4,6 +4,7 @@ create table nflow_workflow (
   id int primary key generated always as identity,
   status varchar(32) not null check (status in ('created', 'executing', 'inProgress', 'finished', 'manual')),
   type varchar(64) not null,
+  priority int not null default 0,
```

Comment:
There is no index changes related to priority field?

Reply:
No, because ORDER BY index will only matter if the executor has to choose from thousands of instances to be executed right now, which is not a case that warrants increasing the size of the index.